sources/osg/WebGLCaps.js

```diff
@@ -103,6 +103,8 @@ WebGLCaps.prototype = {
 
         this._isGL2 = typeof window.WebGL2RenderingContext !== 'undefined' && gl instanceof window.WebGL2RenderingContext;
 
+        Texture = require( 'osg/Texture' );
```

Comment:
was related but maybe not needed anymore.
getting VAO extension existance at Geometry.js parse time was causing circular dependencies
will check if still needed and/or remove when cleaning PR